src/main/java/wolox/training/exceptions/handler/ErrorHandler.java

```diff
@@ -21,6 +26,11 @@ protected ErrorResponse BookNotFoundException(BookNotFoundException ex) {
                 ErrorsEnum.JSON_BOOK_NOT_FOUND_EXCEPTION.getMessage());
     }
 
+    /**
+     *
+     * @param ex: Exception BookIdMismathException
```

Comment:
@MatiasVillalba96 change word BookIdMismathException to BookIdMismatchException.
😄